Addition of subscription information (#905)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/subscriptions-and-billing/what-if-my-subscription-expires.md

If your subscription ends—either because it expires, or because you decide to cancel—your access to Office 365 services, applications, and customer data go through multiple states before the subscription is fully turned off, or *deprovisioned*. If you are aware of this progression, you'll be better equipped to return your subscription to an active state before it's too late, or—if you're leaving Office 365—back up your data before it is ultimately deleted.
  
## Office 365 for business: Subscription lifecycle

> - If your subscription expires, then it goes through following stages: Expired / Disabled / Deprovisioned. The stage “Expired” starts directly after the subscription reach to its end date.

Reviewer: ```suggestion
- If your subscription expires, it goes through the following stages: Expired / Disabled / Deprovisioned. The Expired stage starts directly after the subscription has reached its end date.
```

> - If your subscription expires, it goes through the following stages: Expired / Disabled / Deprovisioned. The Expired stage starts directly after the subscription has reached its end date.

Reviewer 2: - If your subscription expires, it goes through the following stages: Expired, Disabled, and Deprovisioned. The Expired stage starts immediately after the subscription has reached its end date.

> - If you cancel the renewal of your annual subscription, it goes through the same stages as an expired subscription and the first stage starts at the anniversary of the annual subscription (not at the date of cancelling the subscription renewal).

Reviewer 2: - If you turn off recurring billing on your annual subscription, it goes through the same stages as an expired subscription. The first stage starts are the anniversary of the annual subscription, not starting on the date that you turned off the subscription's recurring billing setting.

Author: Suggestions applied.

> - If you cancel the renewal of your annual subscription, then it goes through the same stages as expired subscription and the first stage starts at the anniversary of the annual subscription (not at the date of cancelling the subscription renewal).

Reviewer: ```suggestion
- If you cancel the renewal of your annual subscription, it goes through the same stages as an expired subscription and the first stage starts at the anniversary of the annual subscription (not at the date of cancelling the subscription renewal).
```

> - If you cancel your monthly subscription, then it will be disabled immediately (at the date of cancellation). This means your users will lost the access the Office 365 assets immediately and only admins will get access to the data for next 90 days.

Reviewer: ```suggestion
- If you cancel your monthly subscription, it will be disabled immediately (at the date of cancellation). This means your users will lose access to the Office 365 assets immediately and only admins will have access to the data for the next 90 days.
```

> If you want your subscription data to be deleted before the typical Disabled period is over, you can request expedited deprovisioning. When you request expedited deprovisioning, your subscription data is deleted within 3 days of cancellation. To use expedited deprovisioning, [call support](../contact-support-for-business-products.md).

> [!NOTE]

> > The information on this page is subject to the [Microsoft Policy Disclaimer and Change Notice](https://go.microsoft.com/fwlink/p/?LinkId=613651). Return to this site periodically to review any changes.

Reviewer 2: This edit is unnecessary.

> > The information on this page is subject to the [Microsoft Policy Disclaimer and Change Notice](https://go.microsoft.com/fwlink/p/?LinkId=613651). Return to this site periodically to review any such changes.

Reviewer: ```suggestion
> The information on this page is subject to the [Microsoft Policy Disclaimer and Change Notice](https://go.microsoft.com/fwlink/p/?LinkId=613651). Return to this site periodically to review any changes.
```
